scripts/setup_ubuntu.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+sudo apt update
+sudo apt install -y mongodb
+sudo apt-get install libcairo2-dev
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt update
+sudo apt install yarn nodejs
+sudo mkdir -p /data/db
+sudo chown -R $USER /data/db
+yarn
+cp ../defaultPasswords.json ../passwords.json
+mongod &
```

Comment:
This is an issue that might need to be addressed in `setup.sh` for Mac users (don't worry about that), but `mongod &` still dumps output to the terminal despite the process being run in the background. This can confuse users:

| Terminal Output |
| --- |
| ![image](https://user-images.githubusercontent.com/30734384/57498761-079c8280-7292-11e9-94eb-c2fa3a65ec4f.png) |

(The `Finished.` line appears near the top of the screen, meaning the script finished executing. But it's unclear, since `mongod` decides to drop a bunch of extra lines, overrunning both that and the terminal prompt, which can confuse people into thinking the script is still executing.)

Here's an alternative line that should work:
```bash
mongod > /dev/null 2>&1 &
```

If that doesn't work in the script, then keep this line as is (i.e. leave it as `mongod &`).